### DISCUSSION TITLE
Debian Hyprland theme

### DISCUSSION THREAD

**Discussion by Senshi111** [prasanthrangan/hyprdots#126] [General]
Thanks to prasanthrangan i was able to make those themes compatible with debian sid 
for everyone who wants to try it in debian sid, here is the link [debian hyprdots](https://github.com/Senshi111/debian-hyprland-hyprdots).
Maybe the documentation is still wrong in some places and refers to arch 

I just want to thank prasanthrangan for his excellent work

**Comment by prasanthrangan** [comment]
Thats amazing, thank you so much! Is everything working in debian as expected??
I'll definitely link your repo here in my readme start page!

**Comment by Senshi111** [comment]
Everything works as expected. Only thing that not working that i noticed is clipboard history. To be honest i don't use it so im lazy to make it to work :). I will in future.
here's some screenshots
![230828_21h35m06s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/40372242/91e3046b-38e9-4564-bf68-4a4c5712b52e)
![230828_21h35m33s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/40372242/a778515c-59c6-4997-8bb2-ed669dc1bf75)
![230828_21h36m09s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/40372242/f044372c-887f-4d11-b80b-d8cb6949328a)

**Comment by prasanthrangan** [comment]
Nice, enjoyy... looks like you made it work with nvidia hybrid graphics!!
I've also updated my readme and linked your repo :)

**Comment by Senshi111** [comment]
some small video everything is working fine


https://github.com/prasanthrangan/hyprdots/assets/40372242/8a80c3cf-050d-469d-aa03-7af7b07a771d
